src/language/blockString.js

```diff
@@ -66,10 +66,14 @@ function leadingWhitespace(str) {
   return i;
 }
 
-function isBlank(str) {
+export function isBlank(str: string): boolean {
   return leadingWhitespace(str) === str.length;
 }
 
+export function isPrintableBlockString(str: string): boolean {
+  return Boolean(dedentBlockStringValue(str));
```

Comment:
please explicitly compare with empty string `=== ''`

Reply:
`dedentBlockStringValue` is an expensive function since split strings and does a lot of other manipulations. So it totally ok to use for the draft stage but it should be replaced with a simpler function before merge.